### DISCUSSION TITLE
Horizontal Timeline

### DISCUSSION THREAD

**Discussion by rainbowdog1** [ant-design/ant-design#43180] [Ideas]
As the title says, I would like a horizontal timeline as it can be better incorporated into the website.
Thanks

**Comment by MadCcc** [comment]
Horizontal Timeline would be unable to hold long text.
Or did you mean Steps https://ant.design/components/steps#components-steps-demo-progress-dot ?

**Reply by rainbowdog1** [reply]
The text could be wrapped so from one long line could be two or three, and no steps are another thing
